### PR TITLE
Remove vendor specific references in test files

### PR DESCRIPTION
**Pull request by matthewd0123**
There were a few references to proprietary properties in the test cases. Those have been removed.

### CODE REVIEW

File: tests/test_uri/test_validator/test_urivalidator.py [hunk 1]
```diff
@@ -523,13 +523,13 @@ def test_is_remote_when_authority_is_null(self):
     #     self.assertEqual(authority.number_case(), UAuthority.NumberCase.ID)
 
     # def test_is_remote_when_authority_has_name_and_number_set(self):
-    #     authority = UAuthority(name="vcu.veh.gm.com", id=b"hello Jello")
+    #     authority = UAuthority(name="vcu.veh.com", id=b"hello Jello")
```
Comment (stevenhartley) [inline tests/test_uri/test_validator/test_urivalidator.py:526]: We don't have to remove gm.com references, only the other word :-)

**Review by stevenhartley — CHANGES_REQUESTED** [review]
Please remove on the known vendor specific product name, you can keep gm.com in the URLs

File: tests/test_uri/test_validator/test_urivalidator.py (hunk 1 above)
Comment (matthewd0123) [reply tests/test_uri/test_validator/test_urivalidator.py]: donedone

**Review by stevenhartley — APPROVED** [review]
LGTM :-)

**Review by neelam-kushwah — APPROVED** [review]
LGTM Thanks @matthewd0123

**Pull request merged**